### PR TITLE
C2 amr a raouf c2 s01 ro fb connect

### PR DESCRIPTION
**Pull request by AmrARaouf**
[#355]

@karimngr @Mirna-Yacout @Omar-Hossam 

@SalmaFG  I did not write any model or controller functions so there are no documentations or tests to be reviewed

### CODE REVIEW

**Comment by ghost** [comment]
verified

**Comment by Mirna-Yacout** [comment]
- if condition switched
- if condition should go to model and come back

**Comment by AmrARaouf** [comment]
@Mirna-Yacout please re review

**Comment by Mirna-Yacout** [comment]
verified

File: arability/app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -133,22 +133,37 @@
...
-
-        <div class="Facebook" style="width: 344px;margin-right: auto;margin-left: auto;">
-        </div>
+      <div class="well" style="width:1100px; height:480px; text-align:center; display:inline-block">
+        <div class="providers" style="padding:30px; text-align:center">
+
+          <div class="Facebook" style="padding:10px">
+            <% if Authentication.exists?(gamer_id: current_gamer.id, provider: "facebook") %>
+            <img alt="Facebook" class="border-none" src="/assets/facebook_link.png" width="50px" />
+            <span font='20' style="position:relative; padding:40px;">
```
Comment (omar-mohyeldin) [inline arability/app/views/devise/registrations/edit.html.erb]: use double quotations

File: arability/app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -133,22 +133,37 @@
...
+              <%= link_to t(:remove_connection), "/authentications/remove_connection?provider=#{"facebook"}" %>
+            </span>
+            <% else %>
+            <img alt="Facebook" class="border-none" src="/assets/facebook_link.png" width="50px" height="50px" />
+            <span font='20' style="position:relative; padding:40px;">
```
Comment (omar-mohyeldin) [inline arability/app/views/devise/registrations/edit.html.erb]: use double quotations, and change all single quotations with double quotations for the rest of the code

**Comment by omar-mohyeldin** [comment]
@AmrARaouf almost verified, just please modify this small comment I stated on the code

**Comment by AmrARaouf** [comment]
@Omar-Hossam please re review

File: arability/app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -133,22 +133,37 @@
...
-      <div class="well" style="width:1100px; height:480px">
-
-        <div class="Facebook" style="width: 344px;margin-right: auto;margin-left: auto;">
-        </div>
+      <div class="well" style="width:1100px; height:480px; text-align:center; display:inline-block">
+        <div class="providers" style="padding:30px; text-align:center">
+
+          <div class="Facebook" style="padding:10px">
+            <% if Authentication.exists?(gamer_id: current_gamer.id, provider: "facebook") %>
+            <img alt="Facebook" class="border-none" src="/assets/facebook_link.png" width="50px" />
```
Comment (omar-mohyeldin) [inline arability/app/views/devise/registrations/edit.html.erb]: code should be indented by two spaces after an if condition even if in html code, please modify code's indentation

**Comment by omar-mohyeldin** [comment]
@AmrARaouf I just found another mistake, please modify again

**Comment by AmrARaouf** [comment]
@Omar-Hossam please review again

**Comment by omar-mohyeldin** [comment]
@AmrARaouf code verified

**Pull request merged**